src/prerequestscript.js

```diff
@@ -440,68 +250,13 @@ var AcquiaHttpHmac = function () {
           signature = encodeURI(CryptoJS.HmacSHA256(signature_base_string, this.config.parsed_secret_key).toString(CryptoJS.enc.Base64)),
           authorization = 'acquia-http-hmac ' + authorization_string + ',headers="' + authorization_signed_headers_string + '",signature="' + signature + '"';
 
-        // Set the authorizations headers.
-        request.acquiaHttpHmac = {};
-        request.acquiaHttpHmac.timestamp = x_authorization_timestamp;
-        request.acquiaHttpHmac.nonce = nonce;
-
```

Comment:
I removed these because I don't understand where they get used. Are you using them somewhere later in a test script or such?